unblob/processing.py

```diff
@@ -233,6 +233,10 @@ def calculate_entropy(path: Path, *, draw_plot: bool):
     # We could use the chunk size instead of another syscall,
     # but we rely on the actual file size written to the disk
     file_size = path.stat().st_size
+
+    if file_size == 0:
```

Comment:
I would rather blow up, because this should never happen.

Reply:
Yeah, we should reveal the underlying issue earlier instead. In this case, I found it that we mix the `start_offset` and the `end_offset` of an `UnknownChunk`, so we create and empty file.

I put this PR to draft and do the error handling on the `Chunk` level and fix this issue, then reopen and ping you when it is ready.